### ISSUE TITLE
Help for experiments trained only on Vimeo

### ISSUE THREAD

**Issue by Oceanlib** [danielism97/ST-MFNet#8]
Hi, thanks for your code!
We are trying to build an unified benchmark of most of the SOTA VFI methods. For 2x VFI with four input frames, we see that the training set of FLAVR(https://github.com/tarun005/FLAVR) and VFIT(https://github.com/zhshi0816/Video-Frame-Interpolation-Transformer) are the pure Vimeo (septulets set), while, in ST-MFNet, it is a mixture of Vimeo(septulets set) and BVI. So we retrained the ST-MFNet on the pure Vimeo (septulets set). But, unfortunately the best PSNR we got on validation set of Vimeo (septulets set) is only 35.29, which is quite worse than the model released (36.49 on the same validation set). We think there must be some errors about the parameter settings, but we have tried a few weeks and got no improvement till now. 
So, we are here to seek the help for the complement of our experiments.
Could you train your model on the pure Vimeo (septulets set) and release the model or evaluation results on Vimeo, UCF101, DAVIS and SNU-FILM(4 subsets).
Thanks!

**Comment by danier97** [comment]
Hi,

Thank you for your interest in our work. I will get started with the suggested training and get back to you as soon as I have the results. There might be delays though because I am working on another project at the moment.

Many thanks.

**Comment by Oceanlib** [comment]
Thanks for your reply!

**Comment by danier97** [comment]
Hi, sorry for the delay. I've trained the model on vimeo septuplets training set for 60 epochs and it looks like training has plateaued. You can access the model from [here](https://drive.google.com/file/d/1VWj6pgLc8OE9AItddtRAWWbTE_F132PB/view?usp=sharing). The PSNR on video-septuplets test split is 36.23dB.

Thanks again for your interest in our work.

**Comment by Oceanlib** [comment]
Thanks for your timely help!
